.envrc

```diff
@@ -1,2 +1,3 @@
 use flake
 has just
+export GIT_COMMIT_HASH=$(git rev-parse --short=8 HEAD)
```

Comment:
This will not update when you commit, but okay.

Reply:
It's more to fix our build tools. Another alternative is to have just reload the direnv file on build

Reply:
just setting it to `export GIT_COMMIT_HASH=` is sufficient to make our tools work.